talent/application-process.md

```diff
@@ -4,27 +4,38 @@ This document describes the source{d} hiring process from the point of view of t
+- Less than a week after the submission of the application the candidate receives an email with an update; however there can be delays. In any case, the candiate is free to reach out to us at any moment at talent@sourced.tech.
 
-- In the case of positive feedback in the application, the developer is asked to move to a quick Skype/Zoom/Phone call with the lead of the potential hiring team.
+- In the case of positive feedback in the application, the candidate is invited to a quick video call with the Talent Acquisition Manager.
 
-### Skype/Zoom/Phone call
+### Initial interview: 1st stage
 
-- The developer has a quick call with the lead engineer of the team they applied to. This should be a general discussion that can last between 15-30 minutes. Its purpose is to get to know the background and experience of the developer, as well as allowing the developer to get to know source{d} better. It's expected that the developer has already done some basic research about source{d} and has specific questions about the role and/or the company to be expressed during the call. 
+- The candidate has an introductory video call with the Talent Acquisition Manager. This is a general discussion that can last between 15-30 minutes. This is to allow us to get to know the background and experience of the candidate, as well as allowing them to get to know source{d} better. It's expected that the candidate will have already done some basic research about source{d} and has some specific questions about the role and/or the company to be expressed during the call. 
 
-- There are notable exceptions to this, such as higher volume junior roles where we directly ask the developer to do the coding challenge before a call.
+- There are notable exceptions to this, such as higher volume junior engineering roles where we directly ask the developer to do the coding challenge before a call.
 
-### Coding Challenge
+
+### Hiring Manager interview: 2nd stage
+
+- After the initial interview, the Talent Acquisition Manager will share a summary of the call with the Hiring Manager, and will discuss the suitability of the candidate to the role.
+
+- The Talent Acquisition Manager will contact the candidate to organise the next interview with the hiring manager, or explain why we won't be moving forward with their application
+
+- The Hiring Manager interview will last around one hour via video call, and will be a more indepth conversation about the candidate's experience, their role within the team, how the team works and their future tasks and responsbilities.
+
+### Coding Challenge (Engineering)/ Assignment (Non-Engineering): 3rd stage 
+
+- For Non-engineering roles, candidates will usually be asked to complete an assignment or case study. In some cases this may be presented to members of the team in an interview.
```

Comment:
```suggestion
- For non-engineering roles, candidates will usually be asked to complete an assignment or case study. In some cases this may be presented to members of the team in an interview.
```

Either "non-Engineering" or "non-engineering"